sdk/servicebus/service-bus/samples/typescript/src/useProxy.ts

```diff
@@ -38,14 +40,18 @@ export async function main() {
       // No need to pass the `WebSocket` from "ws" package if you're in the browser
       // in which case the `window.WebSocket` is used by the library.
       webSocket: WebSocket,
-      webSocketConstructorOptions: { agent: proxyAgent }
-    }
+      webSocketConstructorOptions: { agent: proxyAgent },
+    },
+  });
+
+  const sender = sbClient.createSender(queueName);
+
+  console.log(`Sending message using proxy server ${proxyInfo}`);
```

Comment:
Did you mean to leave this in?

Reply:
I did - part of what makes this annoying to detect is that we weren't actually opening the connection at all. I figured send() is a simple enough operation at least.